### PR TITLE
docs: update with examples

### PR DESCRIPTION
**Pull request by kissa96**
Updated documentation with missing example codes

### CODE REVIEW

**Review by FASTSHIFT — DISMISSED** [review]

File: examples/styles/lv_example_style_19.c [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+static lv_style_t style1;
+
+static void event_cb(lv_event_t * e)
+{
+    LV_LOG_USER("Clicked");
+
+    static uint8_t change = 0;
+    lv_obj_t * btn = lv_event_get_target(e);
+    lv_obj_t * obj = lv_obj_get_parent(btn);
+    //Change style properties
```
Comment (kisvegabor) [inline examples/styles/lv_example_style_19.c]: Use `/* */` comments
```suggestion
    /* Change style properties */
```

File: examples/styles/lv_example_style_19.c [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    if(change){
+        lv_style_set_border_side(&style1, LV_BORDER_SIDE_BOTTOM | LV_BORDER_SIDE_RIGHT);
+    } else{
+        lv_style_set_border_side(&style1, LV_BORDER_SIDE_TOP | LV_BORDER_SIDE_LEFT);
+    }
+    change = ~change;
+    //Notify the object, that a property has been changed
```
Comment (kisvegabor) [inline examples/styles/lv_example_style_19.c]: Same here

File: examples/widgets/checkbox/lv_example_checkbox_3.c [hunk 3]
```diff
@@ -0,0 +1,64 @@
+#include "../../lv_examples.h"
+#if LV_USE_CHECKBOX && LV_BUILD_EXAMPLES
+
+static void event_handler0(lv_event_t * e)
+{
+    lv_event_code_t code = lv_event_get_code(e);
+    lv_obj_t* obj = lv_event_get_target(e);
+    lv_obj_t* other_obj = lv_obj_get_child(lv_obj_get_parent(obj), (1 - lv_obj_get_child_cnt(obj)));
+    lv_state_t state = lv_obj_get_state(obj);
```
Comment (kisvegabor) [inline examples/widgets/checkbox/lv_example_checkbox_3.c]: FYI, see the CI error below.

File: examples/widgets/spinbox/index.rst [hunk 4]
```diff
@@ -1,5 +1,5 @@
 
-Simple Spinbox
+Simple Spinbox with cursor style added
```
Comment (kisvegabor) [inline examples/widgets/spinbox/index.rst:2]: The length of `------` under the title should match the length of the title.

File: examples/widgets/switch/lv_example_switch_3.c [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    }
+}
+
+void lv_example_switch_3(void)
+{
+    lv_obj_set_flex_flow(lv_screen_active(), LV_FLEX_FLOW_ROW);
+    lv_obj_set_flex_align(lv_screen_active(), LV_FLEX_ALIGN_CENTER, LV_FLEX_ALIGN_CENTER, LV_FLEX_ALIGN_CENTER);
+
+    lv_obj_t * sw0;
+    lv_style_t* style0;
```
Comment (kisvegabor) [inline examples/widgets/switch/lv_example_switch_3.c]:
```suggestion
    lv_style_t style0;
```

File: examples/widgets/switch/index.rst [hunk 6]
```diff
@@ -5,10 +5,14 @@ Simple Switch
...
 
-
 Switch Orientation
 -------------------
 
 .. lv_example:: widgets/switch/lv_example_switch_2
   :language: c
 
+Adding a style to a swtich
+-------------------
```
Comment (kisvegabor) [inline examples/widgets/switch/index.rst]:
```suggestion
Adding a style to a switch
--------------------------
```

File: examples/widgets/textarea/index.rst [hunk 7]
```diff
@@ -18,3 +18,8 @@ Text auto-formatting
 .. lv_example:: widgets/textarea/lv_example_textarea_3
   :language: c
 
+Textarea placeholder styling
+--------------------
```
Comment (kisvegabor) [inline examples/widgets/textarea/index.rst]:
```suggestion
Textarea placeholder styling
----------------------------
```

Comment (kissa96) [reply examples/widgets/textarea/index.rst]: Fixed in next patch

File: examples/widgets/switch/index.rst (hunk 6 above)
Comment (kissa96) [reply examples/widgets/switch/index.rst]: Fixed in next patch

File: examples/widgets/spinbox/index.rst (hunk 4 above)
Comment (kissa96) [reply examples/widgets/spinbox/index.rst:2]: Fixed in next patch.

File: examples/styles/lv_example_style_19.c (hunk 1 above)
Comment (kissa96) [reply examples/styles/lv_example_style_19.c]: Changed it

File: examples/styles/lv_example_style_19.c (hunk 2 above)
Comment (kissa96) [reply examples/styles/lv_example_style_19.c]: Changed

File: examples/widgets/switch/lv_example_switch_3.c (hunk 5 above)
Comment (kissa96) [reply examples/widgets/switch/lv_example_switch_3.c]: This is outdated. The newest commit has this fixed. CI passes now.

**Comment by kissa96** [comment]
Added new commit with suggested fixes

File: docs/widgets/checkbox.rst [hunk 8]
```diff
@@ -37,10 +37,17 @@ The text can be modified with the
...
 
+.. code-block:: c
+   lv_checkbox_set_text(chechkbox0, "Lorem ipsum"); /*Set the text of checkbox0 to lorem ipsum*/
+
 To set a static text, use :cpp:expr:`lv_checkbox_set_static_text(cb, txt)`. This
 way, only a pointer to ``txt`` will be stored. The text then shouldn't
 be deallocated while the checkbox exists.
 
+.. code-block:: c
+   char[] txt0 = "Reginam occidere...";
```
Comment (kisvegabor) [inline docs/widgets/checkbox.rst]: Just to show a non de-allocated example :slightly_smiling_face:  
```suggestion
   const char * txt0 = "Reginam occidere...";
```

File: examples/widgets/checkbox/lv_example_checkbox_3.c [hunk 9]
```diff
@@ -0,0 +1,63 @@
+#include "../../lv_examples.h"
+#if LV_USE_CHECKBOX && LV_BUILD_EXAMPLES
+
+static void event_handler0(lv_event_t * e)
+{
+    lv_event_code_t code = lv_event_get_code(e);
+    lv_obj_t * obj = lv_event_get_target(e);
+    lv_obj_t * other_obj = lv_obj_get_child(lv_obj_get_parent(obj), (1 - lv_obj_get_child_cnt(obj)));
+    if(code == LV_EVENT_VALUE_CHANGED) {
+        //Handle style changes
```
Comment (kisvegabor) [inline examples/widgets/checkbox/lv_example_checkbox_3.c]:
```suggestion
        /*Handle style changes*/
```
And for the others too.

**Review by kisvegabor — DISMISSED** [review]

File: examples/widgets/checkbox/lv_example_checkbox_3.c [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+
+static void event_handler0(lv_event_t * e)
+{
+    lv_event_code_t code = lv_event_get_code(e);
+    lv_obj_t * obj = lv_event_get_target(e);
+    lv_obj_t * other_obj = lv_obj_get_child(lv_obj_get_parent(obj), (1 - lv_obj_get_child_cnt(obj)));
+    if(code == LV_EVENT_VALUE_CHANGED) {
+        /*Handle style changes*/
+        if(lv_obj_get_state(obj) == LV_STATE_CHECKED) {
+            //If other object is also checked, then trigger an event manually for the 3rd object
```
Comment (kisvegabor) [inline examples/widgets/checkbox/lv_example_checkbox_3.c:12]:
```suggestion
            /*If other object is also checked, then trigger an event manually for the 3rd object*/
```
Check other comments too for consistency

**Comment by kisvegabor** [comment]
Closing in lack of updates.

**Pull request closed without merging**